docs/source/changelog.rst

```diff
@@ -10,6 +10,10 @@ For more detailed information, see `GitHub <https://github.com/jupyter/notebook>
 -----
 
 - Properly handle reaping of zombie subprocesses.
+- Moved the cell toolbar selector into the view menu ![screencast](https://cloud.githubusercontent.com/assets/335567/10711889/59665a5a-7a3e-11e5-970f-86b89592880c.gif).
```

Comment:
Nope, no cool converter, you're right.  I wish we used MD everywhere -- or RST everywhere, including GitHub.  I always forget to change mental contexts.